src/sentry/models/environment.py

```diff
@@ -48,8 +48,28 @@ def get_cache_key(cls, organization_id, name):
         return 'env:2:%s:%s' % (organization_id, md5_text(name).hexdigest())
 
     @classmethod
+    def get_name_or_default(cls, name):
+        return name or ''
+
+    @classmethod
+    def get_for_organization_id(cls, organization_id, name):
+        name = cls.get_name_or_default(name)
+
+        cache_key = cls.get_cache_key(organization_id, name)
+
+        env = cache.get(cache_key)
+        if env is None:
+            env = cls.objects.get(
+                name=name,
+                organization_id=organization_id,
+            )
+            cache.set(cache_key, env, 3600)
```

Comment:
I assume we don't ever need to worry about cache invalidation?

Reply:
Just cargo culting https://github.com/getsentry/sentry/blob/f0814f003bae8e008463d43d34c8b67bb4b65a35/src/sentry/models/environment.py#L82

Reply:
Ah. I see. 🥇

Reply:
As far as I know these can't be edited or deleted (but I don't think that's enforced anywhere either)